playbooks/nodepool-builder.yaml

```diff
@@ -67,11 +67,9 @@
       vars:
         nodepool_task_manager:
           - service
-        # disable nodepool launcher service
-        nodepool_file_nodepool_launcher_service_manage: false
-        nodepool_file_nodepool_launcher_service_config_manage: false
-        nodepool_service_nodepool_launcher_manage: false
-        nodepool_service_nodepool_builder_state: "{{ service_state|default('started') }}"
```

Comment:
Here we delete the var, then where we use it?

Reply:
these variables have default values in the role `ansible-role-nodepool`, and as default, the launcher and builder service are enabled and started state. previously we do not start builder when deploy launcher and do not start launcher when deploy builder. now we have set these value as false in `inventory/group_vars/nodepool.yaml` and enable launcher when deploy launcher, enable builder when deploy builder.

Reply:
@liu-sheng thank you for this clarification.